### PR TITLE
Enable verbose window title text

### PR DESCRIPTION
**Pull request by AltF5**

### CODE REVIEW

**Comment by diversenok** [comment]
Haven't you seen the message that everyone should postpone opening issues and pull requests until *after* the migration? Give the maintainers a break, man, they are already busy.

**Comment by AltF5** [comment]
> Haven't you seen the message that everyone should postpone opening issues and pull requests until _after_ the migration? Give the maintainers a break, man, they are already busy.

This can await until after migrating. But sounded OK https://github.com/winsiderss/systeminformer/issues/1277#issuecomment-1159128440

File: SystemInformer/mainwnd.c
Comment (jxy-s) [inline SystemInformer/mainwnd.c:124]: This could be cleaner.

```suggestion
            PhAppendFormatStringBuilder(&stringBuilder, L" %lu", GetCurrentProcessId());
```

Also looks like the way you had it you might have an extra space when appending if the driver is connected below?

File: SystemInformer/mainwnd.c
Comment (jxy-s) [inline SystemInformer/mainwnd.c:171]: I think it would be most "verbose" (and accurate) to just show the `integrityLevel` string here instead of inferring "(Elevated)"/"(Not Elevated)" like you are.

File: SystemInformer/settings.c
Comment (jxy-s) [inline SystemInformer/settings.c:24]:
```suggestion
    )
```

File: SystemInformer/settings.c
Comment (jxy-s) [inline SystemInformer/settings.c:305]:
```suggestion
    )
```

File: SystemInformer/mainwnd.c
Comment (jxy-s) [inline SystemInformer/mainwnd.c:142]: Clever way to format this info. I think you should clean up and combine it into a format string like I suggested above.

File: SystemInformer/mainwnd.c
Comment (jxy-s) [inline SystemInformer/mainwnd.c:154]: Is there a reason you didn't do this? What you have here seems reasonable to me.

**Review by jxy-s — CHANGES_REQUESTED** [review]

File: SystemInformer/mainwnd.c
Comment (jxy-s) [inline SystemInformer/mainwnd.c:146]:
```suggestion
            PhGetProcessSessionId(NtCurrentProcess(), &sessionID);
```

**Review by jxy-s — CHANGES_REQUESTED** [review]

File: SystemInformer/mainwnd.c
Comment (AltF5) [reply SystemInformer/mainwnd.c:166]: I agree, I can change this

File: SystemInformer/mainwnd.c
Comment (AltF5) [reply SystemInformer/mainwnd.c:149]: Wasn’t sure if this was overkill, or not. I guess there is also some(?) value in seeing what was passed to a CreateProcess[xxx] function for the WinSta\Desktop.

But yes, most accurate to do ObjectNameInformation for the desktop handle.

File: SystemInformer/mainwnd.c
Comment (diversenok) [inline SystemInformer/mainwnd.c:123]: `PhGetOwnTokenAttributes().TokenSid` can be null

File: SystemInformer/mainwnd.c
Comment (diversenok) [inline SystemInformer/mainwnd.c:126]: You're performing a LSA lookup for a SID to just get a string for comparison? `RtlEqualSid` is the way to go.

File: SystemInformer/mainwnd.c
Comment (diversenok) [inline SystemInformer/mainwnd.c:171]: `PhGetProcessIsBeingDebugged`

**Comment by dmex** [comment]
Blocked on #1303

**Comment by jxy-s** [comment]
Please rebase the branch to master. I will be closing the PR if no progress is made.

**Pull request closed without merging**
